drivers/interrupt_controller/Kconfig.scr_ipic

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Syntacore
+# SPDX-License-Identifier: Apache-2.0
+
+config SCR_IPIC
+	bool "Syntacore Integrated Programmable Interrupt Controller (IPIC)"
+	default n
+	select MULTI_LEVEL_INTERRUPTS
+	select 2ND_LEVEL_INTERRUPTS
```

Comment:
This should be `default y` and `depends on DT_HAS_...`. See how this is done on the other IRQ controllers.